gensim/models/nmf.py

```diff
@@ -602,44 +620,57 @@ def update(self, corpus):
                         num_terms=self.num_tokens,
                     )
 
+                    chunk_len = len(chunk)
+
+                logger.info(
+                    "PROGRESS: pass %i, at document #%i/%i",
+                    pass_, chunk_idx * chunksize + chunk_len, lencorpus
```

Comment:
lencorpus may be a float (inf) here, but the formatstring requires it to be an integer